src/Navigation/Education/index.js

```diff
@@ -1,9 +1,27 @@
 import React from "react";
+import "./index.css";
+import EducationContent from "./EducationContent";
+import EducationHeader from "./EducationHeader";
+import data from "../../resume.json";
 function Education(props, ref) {
   return (
-    <h2 ref={ref} className="education">
-      Education
-    </h2>
+    <div ref={ref} className="education">
+      <EducationHeader heading="Education" />
+      <EducationContent
```

Comment:
can use map here instead of writing this block twice, like
`data.education.map(item => <JSX />)`